src/main/java/com/couchbase/lite/storage/SQLiteNativeLibrary.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Created by Pasin Suriyentrakorn.
+ *
+ * Copyright (c) 2015 Couchbase, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package com.couchbase.lite.storage;
+
+import com.couchbase.lite.util.Log;
+import com.couchbase.lite.util.NativeLibraryUtils;
+
+public class SQLiteNativeLibrary {
+    private static final String TAG = Log.TAG_DATABASE;
+
+    // JNI Native libraries:
+    public static final String NATIVE_SQLITE_SYSTEM_LIBRARY = "CBLJavaNativeSQLiteSystem";
+    public static final String NATIVE_SQLITE_CUSTOM_LIBRARY = "CBLJavaNativeSQLite";
+    public static final String NATIVE_SQLCIPHER_LIBRARY = "CBLJavaNativeSQLCipher";
+    public static final String[] NATIVE_LIBRARY_OPTIONS =
+            {NATIVE_SQLCIPHER_LIBRARY, NATIVE_SQLITE_CUSTOM_LIBRARY, NATIVE_SQLITE_SYSTEM_LIBRARY};
+
+    // Use this to override auto loading native libraries:
+    public static String TEST_NATIVE_LIBRARY_NAME = null;
+
+    // JNI Key Derivation Native library:
+    private static final String KEY_LIBRARY = "CBLJavaNativeKey";
+
+    // SQLite library:
+    private static final String ANDROID_SQLITE_LIBRARY = "sqlite";
+    private static final String SQLITE_LIBRARY = "sqlite3";
+
+    // SQLCipher library:
+    private static final String SQLCIPHER_LIBRARY = "sqlcipher";
+
+    public static void load() {
```

Comment:
To simplify the usage, this is the way. Let have a discussion with @zgramana if we would like users to make an API for users to specify.